src/fixup.js

```diff
@@ -297,6 +297,30 @@
     request.send();
   }
 
+  /* Dark mode toggle */
+  const darkCss = document.querySelector('link[href="dark.css"]');
+  if (darkCss) {
```

Comment:
Probably want to check for support... something like: 
```suggestion
  if (darkCss && matchMedia("(prefers-color-scheme)").matches) {
```

Reply:
Suggestion applied in https://github.com/w3c/tr-design/pull/243/commits/2344038765001a652794626b7d41b70c69779e9e